impl/src/main/java/org/arquillian/extension/governor/impl/GovernorTestClassScanner.java

```diff
@@ -145,4 +178,18 @@ private void checkGovernorProviderUniqueness(final Collection<GovernorProvider>
 
         return methodAnnotationsMap;
     }
+
+    private boolean isTestMethod(Method method) {
+        List<String> supportedTestAnnotations = new ArrayList<String>();
+        supportedTestAnnotations.add("org.junit.Test"); // JUnit
```

Comment:
I would put this to enum TestFramework.{JUnit|TestNG} so you would have List<TestFramework> above and in below for cycle you would have if (supportedTestAnnotations.contains(TestFramework.valueOf(annotation.getAnnotationType().getClassName())